aws/cloud-formation-template.yml

```diff
@@ -871,10 +871,11 @@ Resources:
                   --no-cache-dir --compile --ignore-installed \
                   pycurl
                 SLUGIFY_USES_TEXT_UNIDECODE=yes pip3 install \
-                  apache-airflow[celery,postgres,s3]==1.10.2 \
+                  apache-airflow[celery,postgres,s3,crypto]==1.10.2 \
                   celery[sqs] \
                   billiard==3.5.0.4 \
-                  tenacity==4.12.0
+                  tenacity==4.12.0 \
+                  cryptography==2.6.1
```

Comment:
Is there some specific reasion to pin `cryptography`'s version? Is the installation of `apache-airflow[crypto]` bringing a bad version? If not, we can omit this. I'm only pinning billiard and tenacity because they have recent releases that break stuff.